core/tests/protractor_utils/ExplorationEditorMainTab.js

```diff
@@ -502,8 +502,7 @@ var ExplorationEditorMainTab = function() {
     var richTextEditor = await forms.RichTextEditor(stateContentEditor);
     await richTextEditor.clear();
     await richTextInstructions(richTextEditor);
-    expect(await saveStateContentButton.isDisplayed()).toBe(true);
```

Comment:
Thanks for doing this, in general we should avoid having `expect` element inside the page object functions.